mythril/src/ioapic.rs

```diff
@@ -643,8 +643,11 @@ impl From<IoRedTblEntry> for u64 {
 
 #[cfg(test)]
 mod test {
+
+    use log::Level;
+    extern crate testing_logger;
```

Comment:
This might be too annoying to maintain in the long term, but for now I definitely like that we're keeping the same coverage that we have now. Nice work!